h2o-core/src/main/java/water/H2ONode.java

```diff
@@ -145,7 +145,16 @@ public boolean isHealthy(long now) {
   // *interned*: there is only one per InetAddress.
   static private H2ONode intern( H2Okey key ) {
     H2ONode h2o = INTERN.get(key);
-    if( h2o != null ) return h2o;
+    if( h2o != null){
```

Comment:
When the client connects for the first time, this `intern` method is called and `h2o.new UDP_TCP_SendThread()` is started for the new client. When the client disconnects and connects again we need to start the sendThread manually again as it was stopped for the disconnecting client